File: Javaclasswork/GuessTheNumber.java
import java.util.Scanner;
import java.util.Random;
public class GuessTheNumber{
	public static void main(String[] args){
		Scanner input = new Scanner(System.in);
		Random randInt = new Random();
		int randomnumber = randInt.nextInt(1, 1000);
	
int number = 0;
String playAgain = " ";
System.out.println("A random number is generated!");
System.out.println("You have five Trials!!!");

while(!playAgain.equals("no")){

	for(int counter = 1; counter <= 5; counter++){
		
		System.out.println("Guess the number: ");
		number = input.nextInt();
		if (number > randomnumber)
			System.out.println("Too High. Try Again");
		if (number < randomnumber)
			System.out.println("Too Low. Try Again");
		if (number == randomnumber){
			System.out.println("Congratulations. You Guessed The Number!");
			break;
			}
		counter += 1;

		}
	System.out.println("Do you want to keep playing?(Enter yes or no): ") ;
	playAgain = input.next();
	}



	}
}

File: PythonClassWork/guessthenumber.py
from random import randint

counter = 0
random_number = randint(1,1000)
print(random_number)

print('A random number is generated!')
print('You have five Trials!!!')
flag = "yes"
while flag in ["yes"] :

	for number1 in range(5) :
		number = int(input("Guess the number: "))
		if number > random_number:
			print("Too High. Try Again")
		if number < random_number:
			print("Too Low. Try Again")
		if number == random_number:
			print("Congratulations. You Guessed The Number!")

		if counter > 10:
			print("You're Resilient. Well-Done!!!",counter)

		if number1 == 4:
			print("Unfortunately! You've Exhausted your trials!!!")	
		counter += 1
	
			
	flag= input('Do you want to keep playing?(Enter yes or no): ') 

	